Improve and enhance Auth Page & Admin Verification Page

MohammedFaraan opened the pull request:
# ✨ Enhance Auth & Admin Pages with Responsive Design and Mobile Support  

This PR introduces significant improvements and enhancements across the authentication and admin verification workflows to deliver a smoother and more responsive user experience.  

## 🔧 Changes Made  
1. **User Authentication Page** (`src/user/pages/UserAuth.jsx` & `src/user/pages/UserAuth.css`)  
   - Enhanced UI/UX for **Signup/Login**  
   - Refined styling and layout for consistency  

2. **Admin Verification Page** (`src/user/pages/Admin-Verification/Admin.jsx`)  
   - Improved verification flow and interface  

3. **Mobile Enhancements**  
   - Added a dedicated **"Login as Admin"** link for mobile devices on the Auth page  

4. **Responsive Design**  
   - Ensured all pages adapt seamlessly across devices and screen sizes  

## 🏷️ Labels  
- `enhancement`  
- `UI/UX`  
- `responsive-design`  
- `authentication`  
- `OSCI-25`  

---

## Screenshots
<img width="1606" height="754" alt="image" src="https://github.com/user-attachments/assets/f99d2aea-777f-4baa-85c1-e921384dd1ba" />
<img width="316" height="488" alt="image" src="https://github.com/user-attachments/assets/089ccdc1-8784-4cf7-a887-232f80f8d04d" />
<img width="961" height="652" alt="image" src="https://github.com/user-attachments/assets/ad3943a0-b228-4967-9ab6-b4225bfb6b17" />

codervivek5 commented:
kindly mention your issue number ...  because without issue assign we can't accept your PR

MohammedFaraan commented:
Sure.
The issue no.: 2586.
You have assigned it to me yesterday.

I have enhanced the Admin verification page as well, which was not mentioned in the issue.
You can go through pull request description as well.

![IMG_20250820_134917](https://github.com/user-attachments/assets/f1233e3b-339c-491b-8a45-df8c76bd09ee)

codervivek5 commented on src/User/pages/UserAuth/UserAuth.jsx:
```diff
@@ -203,294 +175,311 @@ const AuthForm = () => {
               </button>
-            </div>
+            </Link>
+            <button className="bg-[#f9efe4] text-[#2d3e40] px-6 py-2 rounded-full border border-[#2d3e40] hover:bg-[#2d3e40] hover:text-white transition duration-300">
```
fix this line

codervivek5 commented on src/User/pages/UserAuth/UserAuth.jsx:
```diff
@@ -203,294 +175,311 @@ const AuthForm = () => {
+
+                  <p className="text-xs text-center mt-4">
+                    By continuing, you agree to our{" "}
+                    <Link
+                      to="https://www.vigybag.com/termsAndCondition"
+                      className="text-[#4caf50] hover:underline"
+                    >
+                      Terms of Use
+                    </Link>{" "}
+                    and{" "}
+                    <Link
+                      to="https://www.vigybag.com/privacy"
+                      className="text-[#4caf50] hover:underline"
+                    >
+                      Privacy Policy
+                    </Link>
+                    .
+                  </p>
```
fix this..so when user click on those external page then they can redirect in new tab

codervivek5 commented:
see the coderabbit code then commit...then i'll merge it

MohammedFaraan commented:
> see the coderabbit code then commit...then i'll merge it

Ok

MohammedFaraan commented:
I have updated the code and fix the issues mentioned above.
Following changes done:
1. Removed unwanted button tags within Link tags
2. Changed the incorrect urls of "Terms and conditions" & "Privacy & Policy".
3. Incorrect urls "https://www.vigybag.com/termsAndCondition" & "https://www.vigybag.com/privacy", correct urls "/terms-and-condition" & "/privacy-policy"

MohammedFaraan commented:
I have done a lot of changes and enhancements other than things mentioned in the issue.

So, give a **Level 3** tag for this.
